code/modules/food_and_drinks/food/snacks_other.dm

```diff
@@ -761,3 +761,698 @@
+	name = "jerky wraps"
+	desc = "Thin slices of flavorful beef jerky, carefully wrapped to create a portable and protein-packed snack. Ideal for satisfying your hunger on the go."
+	list_reagents = list(/datum/reagent/consumable/nutriment = 5)
+	filling_color = "#b82121"
+	tastes = list("dry" = 1, "jerky" = 1, "beef" = 1)
+	foodtype = MEAT
+
+/obj/item/reagent_containers/food/snacks/ration/cinnamon_bun
+	name = "cinnamon bun"
+	desc = "A delectable pastry swirled with cinnamon and drizzled with a sweet glaze. Warm and fluffy, this cinnamon bun is a delightful treat to enjoy with your favorite beverage."
+	list_reagents = list(/datum/reagent/consumable/nutriment = 3, /datum/reagent/consumable/sugar = 3)
+	filling_color = "#b82121"
+	tastes = list("cinnamon" = 3, "airy" = 1, "sweet" = 1)
+	foodtype = GRAIN | SUGAR
+
+/obj/item/reagent_containers/food/snacks/ration/pork_spaghetti
+	name = "spaghetti with pork and sauce"
+	desc = "A hearty dish of spaghetti with tender pork and a savory sauce. A filling and delicious meal to satisfy your hunger."
+	list_reagents = list(/datum/reagent/consumable/nutriment = 8)
+	filling_color = "#b82121"
+	tastes = list("pork" = 1, "spaghetti" = 1, "sauce" = 1)
+	foodtype = MEAT | GRAIN | VEGETABLES
+
+/obj/item/reagent_containers/food/snacks/ration/toaster_pastry
+	name = "chocolate chip toaster pastry"
+	desc = "A delicious chocolate chip toaster pastry, perfect for a quick breakfast or a tasty snack. Indulge in the delightful blend of chocolate and pastry."
+	list_reagents = list(/datum/reagent/consumable/nutriment = 5, /datum/reagent/consumable/sugar = 5)
+	filling_color = "#b82121"
+	tastes = list("chocolate" = 1, "pastry" = 1, "sweet" = 1)
+	foodtype = SUGAR | GRAIN | JUNKFOOD | BREAKFAST
+
+/obj/item/reagent_containers/food/snacks/ration/bread_sticks
+	name = "seasoned bread sticks"
+	desc = "Crunchy and flavorful seasoned bread sticks, a delightful accompaniment to your meal or a satisfying snack on their own."
+	list_reagents = list(/datum/reagent/consumable/nutriment = 5)
+	filling_color = "#b82121"
+	tastes = list("bread" = 1, "seasoned" = 1)
+	foodtype = GRAIN
+
+/obj/item/reagent_containers/food/snacks/ration/dried_raisins
+	name = "dried raisins"
+	desc = "Sweet and chewy dried raisins, a natural and healthy snack option. Packed with natural sugars and nutrients for a burst of energy."
+	list_reagents = list(/datum/reagent/consumable/nutriment = 5, /datum/reagent/consumable/sugar = 3)
+	filling_color = "#b82121"
+	tastes = list("raisins" = 1, "sweet" = 1)
+	foodtype = FRUIT | SUGAR
+
+/obj/item/reagent_containers/food/snacks/ration/fried_fish
+	name = "fried fish chunks"
+	desc = "Crispy and delicious fried fish chunks, perfect for seafood lovers. Satisfy your cravings with this delightful fried treat."
+	list_reagents = list(/datum/reagent/consumable/nutriment = 5)
+	filling_color = "#b82121"
+	tastes = list("fish" = 1, "fried" = 1)
+	foodtype = FRIED
```

Comment:
This should be SEAFOOD as well.

Reply:
small issue. Seafood is not real! i can add it but i also need the taste preferences all the species should have for it

Reply:
That seems like a seperate pr tbh